src/elements/content-sidebar/activity-feed/comment/__tests__/Comment-test.js

```diff
@@ -419,4 +419,50 @@ describe('elements/content-sidebar/ActivityFeed/comment/Comment', () => {
         expect(wrapper.find('InlineEdit').length).toEqual(2);
         expect(wrapper.find('UserLink').length).toEqual(1);
     });
+
+    test('should render messageHeader if prop is passed in', () => {
+        const comment = {
+            created_at: TIME_STRING_SEPT_27_2017,
+            tagged_message: 'test',
+            created_by: { name: '50 Cent', id: 10 },
+        };
+        const messageHeader = <div>message header</div>;
+
+        const wrapper = shallow(
+            <Comment
+                id="123"
+                {...comment}
+                approverSelectorContacts={approverSelectorContacts}
+                currentUser={currentUser}
+                handlers={allHandlers}
+                mentionSelectorContacts={mentionSelectorContacts}
+                messageHeader={messageHeader}
+            />,
+        );
+
+        expect(wrapper).toMatchSnapshot();
```

Comment:
Seems like we could make these more valuable by testing for the existence of the markup we expect.